packages/packages-groonga-org-package-task.rb

```diff
@@ -175,19 +175,32 @@ def download_packages(target_namespace)
     mkdir_p(repositories_dir)
     download_dir = "#{base_dir}/tmp/downloads/#{@version}"
     mkdir_p(download_dir)
+    
+    github_releases_base_url = "https://github.com/groonga/groonga/releases/download"
+    htaccess_path = "#{repositories_dir}/.htaccess"
+
+    if (target_namespace == :windows)
+      htaccesss_value = "RewriteRule groonga-((1[2-9]|[2-9][0-9])[\\d\\.]+?)-.+?-vs(?!(2012|2013|2015|2017)).+?\\.zip"\
+                        " #{github_releases_base_url}/v$1/$0\n"
```

Comment:
Do we need this rewrite rule? I think that we only need a link to `-latest`.

Reply:
I had written the reason why this ReqiteRule is needed on description.

> The regexp of the first line is for 12.0.0+ (but not latest) and vs2019+.
> If we omit this, for example, after Groonga 12.1.0 is released, we won't be able to download old versions by version specifying URL like https://packages.groonga.org/windows/groonga/groonga-12.0.9-x64-vs2019-with-vcruntime.zip because v12.0.9 will not exist on packages.groonga.org.

But if we do not need to provide a way to donwnload old version Groonga from packages.groonga.org, it is needless.

Reply:
Thanks. I see.

In my opinion,  this is needless.
Because a download URL that user can know is only https://packages.groonga.org/windows/groonga/groonga-latest-xxx.zip normally.

In addition, I would like to use in the user the latest version when Groonga up to version.
Because the user doesn't need to use a specific version in when Groonga up to version generally.

Reply:
Ok, I will remove this.

Reply:
Removed this `RewriteRule`.